<ui version="4.0" >
 <class>StatReportF131RaionSetupDialog</class>
 <widget class="QDialog" name="StatReportF131RaionSetupDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="lblBegDate" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Дата &amp;начала периода</string>
     </property>
     <property name="buddy" >
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="CDateEdit" name="edtBegDate" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lblEndDate" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Дата &amp;окончания периода</string>
     </property>
     <property name="buddy" >
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="CDateEdit" name="edtEndDate" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="lblEventType" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Тип обращения</string>
     </property>
     <property name="buddy" >
      <cstring>cmbEventType</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="CRBComboBox" name="cmbEventType" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QCheckBox" name="chkOnlyPayedEvents" >
     <property name="text" >
      <string>Только опла&amp;ченные</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="lblBegPayDate" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Оплачено в период с</string>
     </property>
     <property name="buddy" >
      <cstring>edtBegPayDate</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="CDateEdit" name="edtBegPayDate" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="lblEndPayDate" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>по</string>
     </property>
     <property name="buddy" >
      <cstring>edtEndPayDate</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="CDateEdit" name="edtEndPayDate" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Район</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <widget class="QComboBox" name="cmbRaion" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Прикрепление к</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" >
    <widget class="QComboBox" name="cmbPermanentAttach" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="8" column="0" >
    <widget class="QLabel" name="lblYearForDD" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Префикс формы 131</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" >
    <widget class="QLineEdit" name="ledtYearForDD" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maxLength" >
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>cmbEventType</tabstop>
  <tabstop>chkOnlyPayedEvents</tabstop>
  <tabstop>edtBegPayDate</tabstop>
  <tabstop>edtEndPayDate</tabstop>
  <tabstop>cmbRaion</tabstop>
  <tabstop>cmbPermanentAttach</tabstop>
  <tabstop>ledtYearForDD</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StatReportF131RaionSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>125</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel" >
     <x>70</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StatReportF131RaionSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>238</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel" >
     <x>279</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOnlyPayedEvents</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtEndPayDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>296</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel" >
     <x>296</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOnlyPayedEvents</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtBegPayDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>296</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel" >
     <x>296</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
